libr/debug/debug.c

```diff
@@ -863,6 +870,9 @@ R_API int r_debug_step_over(RDebug *dbg, int steps) {
 		if (op.type == R_ANAL_OP_TYPE_CALL  ||
```

Comment:
as long as this is a check for all the type of calls why not use (op.type & R_ANAL_OP_TYPE_CALL == R_ANAL_OP_TYPE_CALL)